ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/runs/$runIndex/issues/index.tsx

```diff
@@ -74,30 +78,27 @@ const columns = [
     filterFn: (row, _columnId, filterValue): boolean => {
       return filterValue.includes(row.original.severity);
     },
-  }),
-  columnHelper.display({
-    id: 'package',
-    header: 'Package ID',
-    cell: function CellComponent({ row }) {
-      if (!row.original.identifier) {
-        return null;
-      }
-      const { type, namespace, name, version } = row.original.identifier;
-      return (
-        <div className='font-semibold'>
-          {type ? type.concat(':') : ''}
-          {namespace ? namespace.concat('/') : ''}
-          {name ? name : ''}
-          {version ? '@'.concat(version) : ''}
-        </div>
-      );
+    sortingFn: (rowA, rowB) => {
+      return compareSeverity(rowA.original.severity, rowB.original.severity);
     },
   }),
+  columnHelper.accessor(
+    (issue) => {
+      return identifierToString(issue.identifier);
```

Comment:
This is used here, but it's defined in a later commit, so the code is broken in this commit.

Reply:
This is now failing for me completely: I have once again the situation, where the commits are in correct order in GB, but still in wrong order in GitHub. Do I need to restart the PR again?